nuxeo-features/nuxeo-annotation/nuxeo-annotation-core/src/test/java/org/nuxeo/ecm/annotation/TestAnnotationService.java

```diff
@@ -101,11 +100,13 @@ public void testGetAnnotation() {
         DocumentModel docToAnnotate = session.createDocumentModel("/", "testDoc", "File");
         DocumentModel annotationModel = session.createDocumentModel(null, "testName", ANNOTATION_DOC_TYPE);
 
+        String annotationId = "xxx";
```

Comment:
Just to avoid confusion with `XXX` comment annotations for important things, please use something else, like `xyz` or `foo` :)